#!/bin/bash -e
pkg=tinylogin-1.4
tar xjf ${1}/${pkg}.tar.bz2
cd ${pkg}
patch -p1 <<"EOF"
--- a/Config.h	2002-06-22 22:09:07.000000000 -0500
+++ b/Config.h	2006-12-18 15:06:12.716030322 -0600
@@ -9,10 +9,10 @@
 #define CONFIG_DELGROUP
 #define CONFIG_GETTY
 #define CONFIG_LOGIN
-#define CONFIG_PASSWD
+//#define CONFIG_PASSWD
 #define CONFIG_SU
-#define CONFIG_SULOGIN
-#define CONFIG_VLOCK
+//#define CONFIG_SULOGIN
+//#define CONFIG_VLOCK
 //
 //
 //
@@ -25,10 +25,10 @@
 #define CONFIG_FEATURE_SHADOWPASSWDS
 //
 // Enable checking of /etc/securetty by login
-#define CONFIG_FEATURE_SECURETTY
+//#define CONFIG_FEATURE_SECURETTY
 //
 // Enable using sha passwords
-#define CONFIG_FEATURE_SHA1_PASSWORDS
+//#define CONFIG_FEATURE_SHA1_PASSWORDS
 //
 // Enable use of a wheel group
 #define CONFIG_WHEEL_GROUP
@@ -41,7 +41,7 @@
 // to create links (or symlinks) at runtime for all the 
 // commands that are compiled into the binary.  This needs 
 // the /proc filesystem to work properly...
-#define CONFIG_FEATURE_INSTALLER
+//#define CONFIG_FEATURE_INSTALLER
 //
 //
 //---------------------------------------------------
EOF
echo "Package ${pkg} patched."
